src/App.tsx

```diff
@@ -1,23 +1,33 @@
-import React from 'react';
+import React, { FC, ReactElement } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import userManager, {
+  loadUser,
+  signinRedirect,
+  signoutRedirect
+} from './auth/user-service';
+import AuthProvider from './auth/auth-provider';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import SignOutOidc from './auth/SignOutOidc';
+import SignInOidc from './auth/SignInOidc';
+import TestAuth from './auth/test-auth/TestAuth';
 
-function App() {
+const App: FC<{}> = (): ReactElement => {
+  loadUser();
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <button onClick={() => signinRedirect()}>Login</button>
```

Comment:
Не пробував signinSilent або signinPopup

Reply:
Так?
```suggestion
        <button onClick={() => signinSilent()}>Login</button>
```
Я это видел, но решил уже сделать с базовой страницей входа, а потом уже менять и экспериментировать
Но это вроде не то, что нам надо